### DISCUSSION TITLE
WordPress Site header.php Issue

### DISCUSSION THREAD

**Discussion by sameelbutt** [community/community#111257] [Projects and Issues]
### Select Topic Area

Question

### Feature Area

Issues

### Body

My [site](https://greenryenthusiast.com/my-electric-lawn-mower-gets-wet/#google_vignette) is made on WordPress and the website uses the Kadence theme and website covers the topic of electric lawnmowers but whenever I install the free plugins it's not properly installed on my site. I also checked my header.php file and some pieces of code are missing in it. Please guide me

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by amankr1469** [comment]
First, deactivate all plugins and then try uploading a Plugin Zip File. Some plugins are not available in the directory.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @sameelbutt ! 

The Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. 

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on WordPress like [their community](https://wordpress.org/support/forums/). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
